appveyor/crowdinSync.py

```diff
@@ -0,0 +1,55 @@
+# based on file from https://github.com/jcsteh/osara
+
+import time
+import os
+import glob
+import sys
+import zipfile
+import shutil
+import subprocess
+import requests
+
+AUTH_TOKEN = os.getenv("crowdinAuthToken")
+if not AUTH_TOKEN:
+	raise ValueError("crowdinAuthToken environment variable not set")
+PROJECT_ID = int(os.getenv("crowdinProjectID"))
+if not PROJECT_ID:
+	raise ValueError("crowdinProjectID environment variable not set")
+
+
+def request(path, method=requests.get, headers={}, **kwargs):
+	headers["Authorization"] = "Bearer %s" % AUTH_TOKEN
+	r = method("https://api.crowdin.com/api/v2/%s" % path, headers=headers,
+		**kwargs)
+	# Convert errors to exceptions, but print the response before raising.
+	try:
+		r.raise_for_status()
+	except requests.exceptions.HTTPError:
+		print(r.json())
+		raise
+	return r
+
+def projectRequest(path, **kwargs):
```

Comment:
PEP 8 suggests snake case for function names
```suggestion
def project_request(path, **kwargs):
```

Reply:
An exception to this is where camel case is already the prevailing style - [PEP 8 link](https://peps.python.org/pep-0008/#function-and-variable-names).

This is the case with NVDA, I believe part of the reason for this is it is synthesized in a more readable manner, perhaps a blind developer can confirmer here.
See our [coding standards](https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/codingStandards.md#identifier-names).

Reply:
Thanks! Withdrawing comment.